Capri.Web/src/components/promotersList.vue

```diff
@@ -0,0 +1,227 @@
+					</template>
+
+					<template v-slot:header.masterTopics="{ header }">
+						<span>{{ header.text }}</span>
+					</template>
+
+					<template v-slot:item.title="{ item }">
+						<span>
+							{{ item.promoter }}
+						</span>
+					</template>
+					<template v-slot:item.bachelorsTopics="{ item }">
+						<span>
+							{{ item.bachelorThesis }}
+						</span>
+					</template>
+					<template v-slot:item.masterTopics="{ item }">
+						<span>
+							{{ item.masterThesis }}
+						</span>
+					</template>
+				</v-data-table>
+			</v-col>
+		</v-row>
+
+		<v-btn id="exportDataButton">EXPORT TO EXCEL</v-btn>
+	</v-container>
+</template>
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import popUp from './popUp.vue';
+
+@Component({
+    components: {
+        popUp,
+    },
+})
+export default class MyProporsals extends Vue {
+    public popUp = {
+        show: false,
+        maxWidth: 600,
+        data: {
+            title: {
+                text: '',
+                label: 'Promoter name',
+                type: 'editableTextField',
+                columns: 12,
+            },
+            promoter: {
+                text: '',
+                label: 'Institute name',
+                items: ['Bioinformatics', 'Robotics'],
+                chosen: '',
+                type: 'selectableField',
+                columns: 12,
+            },
+            thesisType: {
+                text: '',
+                label: 'Degree',
+                items: ['Bachelor', 'Master'],
+                chosen: '',
+                type: 'selectableField',
+                columns: 12,
+            },
+        },
+    };
+    public data() {
+        return {
+            degrees: ['Bachelor', 'Master'],
+            institutes: ['Bioinformatics'],
+            dialog: '',
+            search: '',
+            popUp: {
+                name: '',
+                insitute: '',
+                degree: '',
+            },
+            headers: [
+                {
+                    sortable: false,
+                    text: 'Name',
+                    value: 'title',
+                    width: '40%',
+                    align: 'center',
+                },
+                {
+                    sortable: false,
+                    text: 'Bachelor topics',
+                    value: 'bachelorsTopics',
+                    width: '20%',
+                    align: 'center',
+                },
+                {
+                    sortable: false,
+                    text: 'Master topics',
+                    value: 'masterTopics',
+                    width: '20%',
+                    align: 'center',
+                },
+                {
+                    sortable: false,
+                    value: 'empty',
+                },
+            ],
+            items: [
+                {
+                    promoter: 'Dakota Rice',
+                    bachelorThesis: 3,
```

Comment:
Now for PromoterViewModel.

Rename/change fields:
promoter -> Id (Guid)
bachelorThesis -> ExpectedNumberOfBachelorProposals
masterThesis -> ExpectedNumberOfMasterProposals

Add fields:
TitlePrefix (string)
TitlePostfix (string)
FirstName (string)
LastName (string)
Proposals (ICollection<Guid>)
UserId (Guid)
InstituteId (Guid)

Remove fields:
type
available